data/teams.json

```diff
@@ -0,0 +1,185 @@
+[
+  {
+    "name": "Aftermath",
+    "abbrevation": "aM",
+    "players": [
+      "Hearttt",
+      "MbL",
+      "Nicov"

```

Comment:
I'm thinking of a better way to represent players here. As this is kind of a common known term, but from databases we have the unique identifiers. Maybe to write a small script that enumerates the players in the players.yaml and put in a unique ID here would be good? Or what's your opinion about that @happyleavesaoc? Might be too complicated then to make PRs in the future and add new players. Hmm.

Reply:
Ideally we use the `liquipedia` player property and the Liquipedia API to write a script that automatically generates this file.

Reply:
Good idea!